mmv1/products/dataplex/Datascan.yaml

```diff
@@ -413,6 +441,47 @@ properties:
         name: 'rowFilter'
         description: |
           A filter applied to all rows in a single DataScan job. The filter needs to be a valid SQL expression for a WHERE clause in BigQuery standard SQL syntax. Example: col1 >= 0 AND col2 < 10
+      - !ruby/object:Api::Type::NestedObject
+        name: 'postScanActions'
+        description: |
+          Actions to take upon job completion.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'bigQueryExport'
+            description: |
+              If set, results will be exported to the provided BigQuery table.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'resultsTable'
+                description: |
+                  The BigQuery table to export DataProfileScan results to.
+                  Format:projects/{project}/datasets/{dataset}/tables/{table}
```

Comment:
Is it possible to test this field. It's okay if not just want to ensure we are conscious in it's exclusion.

Reply:
It is required field and has been tested.